addresses.go

```diff
@@ -4,21 +4,20 @@ import "github.com/MOHC-LTD/shopify"
 
 // AddressDTO represents a Shopify Address in HTTP requests and responses
 type AddressDTO struct {
-	Address1     string  `json:"address1,omitempty"`
```

Comment:
I would leave this one alone where it's always been. Customers have a specific address endpoint so the DTO could be reused from that for standard customer ones.

Reply:
![Screenshot 2022-10-17 at 15 38 28](https://user-images.githubusercontent.com/63408186/196206331-2514755a-b14b-4cfa-88ca-5121619d30e7.png)